polaris-server.yml

```diff
@@ -105,6 +105,8 @@ authenticator:
 #    type: symmetric-key
 #    secret: polaris
 
+authorizer: org.apache.polaris.service.auth.CoreAuthorizer
```

Comment:
This should be `org.apache.polaris.core.auth.CoreAuthorizer` – this is why the regtests are failing.

That said I do think that moving it to `polaris-service` would be better :-)